### ISSUE TITLE
Improve accuracy of detecting poses where limbs are occluded

### ISSUE THREAD

**Issue by monsieurpooh** [google/mediapipe#4868]
### MediaPipe Solution (you are using)

Pose

### Programming language

Python

### Are you willing to contribute it

No

### Describe the feature and the current behaviour/state

Pose detection only works well when user is standing and facing the camera. When turning around, performing martial arts techniques, or getting in sitting or prone positions, accuracy suffers

### Will this change the current API? How?

No

### Who will benefit with this feature?

Tons of use cases for games that require poses other than typical upright poses.

### Please specify the use cases for this feature

For example, a fighting game that requires complex movements.

**Comment by mcdonasd1212** [comment]
Agree,  occluded limb detection is very poor and there is a need for an improved solution.

**Comment by kuaashish** [comment]
@monsieurpooh,

Thank you for your thought to extending the functionality of pose landmark. We are forwarding request internally and based on discussion, team will prioritise it's implementation.
